src/core/migrations/0088_fix_untyped_galleys.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2.11 on 2024-04-30 17:04
+
+from django.db import migrations
+from production.logic import get_galley_label_and_type
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0087_merge_20240315_1649'),
+        ('metrics', '0010_auto_20230317_1534'),
+    ]
+
+def fix_untyped_galleys(apps, schema_editor):
+    """ Fixes galleys that haven't had a type set and linked metrics records"""
+
+    Galley = apps.get_model('core', 'Galley')
+    ArticleAccess = apps.get_model('metrics','ArticleAccess')
+    for galley in Galley.objects.filter(type="",file__isnull=True, article__isnull=True):
```

Comment:
Is this supposed to exclude Galleys that have files/articles?

Reply:
OK yes I have this question too.